smg-automotive/components-pkg
# fix: disable redirect to home page on language change

References [DM-1982](https://autoricardo.atlassian.net/browse/DM-1982)

## Motivation and context

On language change user should stay on the same page

## Before

User is redirected to home page

## After

User stays on the current page

## How to test

- Go to https://test-fix-url-on-language-change-listings-web.branch.autoscout24.dev/en/s
- Open language menu inside header
- Change language to e.g. fr or it
- User should stay on the same page (SRP)


[DM-1982]: https://autoricardo.atlassian.net/browse/DM-1982?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
